features/steps/test_store_resource.py

```diff
@@ -71,15 +70,18 @@ def message_bus(path_data: PathData, unit_of_work: AbstractUnitOfWork) -> Memory
     translocator = Translocator(
         inbox_path=path_data.inbox,
         workspaces_path=path_data.workspaces,
-        minter = lambda: value
+        minter = lambda: value,
+        file_provider=FilesystemFileProvider()
     )
 
     handlers: dict[Type[Event], list[Callable]] = {
         PackageSubmitted: [
             lambda event: receive_package(event, unit_of_work, translocator)
         ],
         PackageReceived: [
-            lambda event: verify_package(event, unit_of_work, BagAdapter, Workspace)
+            lambda event: verify_package(
+                event, unit_of_work, BagAdapter, Workspace, FilesystemFileProvider()
+            )
```

Comment:
We also need to update this handler in `dor/cli/repo.py`: https://github.com/mlibrary/dor-py/blob/898e97d65381f1af41376c509cc5d73676438962/dor/cli/repo.py#L54